packages/tx/src/eip2930Transaction.ts

```diff
@@ -157,7 +157,7 @@ export default class AccessListEIP2930Transaction extends BaseTransaction<Access
     this.chainId = chainId ? new BN(toBuffer(chainId)) : this.common.chainIdBN()
     this.gasPrice = new BN(toBuffer(gasPrice === '' ? '0x' : gasPrice))
 
-    this._validateCannotExceedMaxInteger({ gasPrice: this.gasPrice })
+    this._validateCannotExceedMaxInteger({ r: this.r, s: this.s, gasPrice: this.gasPrice })
```

Comment:
Have to check but is this not validated in `BaseTransaction`?

Reply:
Is checked; https://github.com/ethereumjs/ethereumjs-monorepo/blob/a1e7a34d989c7991454c06567573608f33caecbf/packages/tx/src/baseTransaction.ts#L69

Reply:
Yes, I moved this from `EIP2930Transaction` to `BaseTransaction` some commits later on. 😄